tutorials/time-series-modeling/index.md

```diff
@@ -0,0 +1,471 @@
+  .arrayFlatten([independents]);
+```
+
+The image added to the map is a two-band image in which each pixel contains
+values for $\beta_0$ and  $\beta_1$. Click around the map with the
+inspector, and look at some of the values. We can see that most pixels around
+our region of interest have a negative trend—although darker values
+indicate a shallow negative trend, while bright red pixels indicate a steeper
+descent.
+
+Use the model to "detrend" the original NDVI time series. By detrend, we mean
+account for the slope of the chart and remove it from the original data.
+
+```js
+// Compute a detrended series.
+var detrended = filteredLandsat.map(function(image) {
+  return image.select(dependent).subtract(
+    image.select(independents).multiply(coefficients).reduce('sum'))
+    .rename(dependent)
+    .copyProperties(image, ['system:time_start']);
+});
+
+var detrendedChart = ui.Chart.image.series(detrended, roi, null, 30)
+  .setOptions({
+    title: 'Detrended Landsat Time Series at ROI',
+    lineWidth: 1,
+    pointSize: 3,
+  });
+print(detrendedChart);
+```
+
+Compared to our earlier graph, the data looks similar—but now, the slight
+downward slope is accounted for with our linear model. Each fitted data point
+(data point on the linear model) is subtracted from each of the observed data
+points. Additionally, the Y-axis is now centered at 0, and the scale ranges
+from 0 to +/- 0.45 This allows us to focus on cyclical patterns in the data
+with long-term trends in the data removed.
+
+![Detrended Data](im_05_04.png)
+
+## Estimate seasonality with a harmonic model
+
+Consider the following linear model, where $e_t$ is random error, $A$ is
+amplitude, $\omega$ is frequency, and $\phi$ is phase:
+
+$$
+p_t = \beta_0 + \beta_1t + Acos(2\pi\omega t - \phi) + e_t
+$$
+
+We can decompose our function into separate cosine and sine elements.
+
+$$
+p_t = \beta_0 + \beta_1t + \beta_2cos(2\pi\omega t) + \beta_3sin(2\pi\omega t) + e_t
+$$
+
+Note that $\beta_2 = Acos(\phi)$ and $\beta_3 = Asin(\phi)$, implying
+$A = (\beta_2^2 + \beta_3^2)^½$ and $\phi = atan(\frac{\beta_3}{\beta_2})$).
+
+In simpler terms, we are breaking up more complex curves into a set of
+simplified cosine waves with an additive term. Mark Jakubauskas has an
+informative
+[paper](https://www.isprs.org/proceedings/xxxiii/congress/part4/384_xxxiii-part4.pdf)
+that breaks down the process, and there are many resources that
+elaborate more on the math behind harmonic models. Building a harmonic model
+is used in remote sensing applications because of its flexibility in
+accounting for cyclicality with simple, reproducible shapes. If there is a
+seasonal trend in the data, the ordered nature of a cosine curve can likely
+approximate it.
+
+To fit this model to the time series, set $\omega$=1 (one cycle per unit
+time) and use ordinary least squares regression as the metric of error
+reduction.
+
+First, add the harmonic variables (the third and fourth terms of equation 2)
```

Comment:
Can you please label all of the equations with e.g., `\tag{1}`, inside of the `$$...$$` sections.

Reply:
I updated the three equations that took up a line with this format - tag to the left. Let me know if there's another way to format it.
$$ \tag{1} y = \beta_0 + \beta_1X_1 + ... + \beta_nX_n + \epsilon_t $$

Reply:
- I believe the convention is on the right. I've made the change in [this commit](https://github.com/google/earthengine-community/pull/446/commits/925679ef25abd888b87324507b3ec9ac9dcbb9db).
- The formula formatter for the Google Developer pages should render the tag properly.